src/ast/passes/codegen_llvm.cpp

```diff
@@ -3292,13 +3304,11 @@ void CodegenLLVM::generateWatchpointSetupProbe(
     int arg_num,
     int index)
 {
-  Function *func = Function::Create(func_type,
-                                    Function::ExternalLinkage,
-                                    get_watchpoint_setup_probe_name(
-                                        expanded_probe_name),
-                                    module_.get());
-  func->setSection(
-      get_section_name_for_watchpoint_setup(expanded_probe_name, index));
+  auto func_name = get_function_name_for_watchpoint_setup(expanded_probe_name,
+                                                          index);
+  Function *func = Function::Create(
+      func_type, Function::ExternalLinkage, func_name, module_.get());
+  func->setSection("s_" + func_name);
```

Comment:
missed one ;)

Reply:
Many thanks for checking and fixing my careless work! Should be fixed now.